### PR TITLE
[ACS-7905] ADF Core lib is missing dependency declaration

### PR DESCRIPTION
**Pull request by dominikiwanekhyland**
**Please check if the PR fulfills these requirements**

> - [ ] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)

https://hyland.atlassian.net/browse/ACS-7905

**What is the new behaviour?**
https://bundlephobia.com/package/@alfresco/adf-core@7.0.0-alpha.7-12081671557

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: lib/core/package.json
Comment (AleksanderSklorz) [inline lib/core/package.json:1]: Do we need these changes also for others libraries like content-services? I see there are different versions too

File: lib/core/package-lock.json
Comment (AleksanderSklorz) [inline lib/core/package-lock.json:1]: Could you please check if you need that file? I guess we don't need to commit that file as it was never committed

File: lib/core/ng-package.json
Comment (AleksanderSklorz) [inline lib/core/ng-package.json:1]: Similarly here like for package.json - we have ng-package.json files also in others libraries so do we need some changes for them as well to ensure people can use them?

File: lib/core/package.json
Comment (dominikiwanekhyland) [reply lib/core/package.json:1]: @AleksanderSklorz the others are not broken and can be checked eg: 
https://bundlephobia.com/package/@alfresco/adf-content-services@7.0.0-alpha.6
https://bundlephobia.com/package/@alfresco/adf-process-services@7.0.0-alpha.6

so I guess we don't need any changes there?

Comment (AleksanderSklorz) [reply lib/core/package.json:1]: ok then fine

**Review by AleksanderSklorz — APPROVED** [review]

**Review by nikita-web-ua — APPROVED** [review]

**Review by swapnil-verma-gl — APPROVED** [review]

**Pull request merged**